suite-native/module-dev-utils/src/screens/DemoScreen.tsx

```diff
@@ -80,7 +80,8 @@ export const DemoScreen = () => {
                     >
                         {transactionNotifications.map(notification => (
                             <TransactionNotification
-                                {...notification}
+                                // @ts-expect-error
```

Comment:
Shouldn't be needed. Just type `transactionNotifications` and get rid of this expect-error.

Reply:
removed in 53e695678